client/src/components/Post/PostList.tsx

```diff
@@ -1,54 +1,42 @@
 import { useDispatch, useSelector } from 'react-redux';
 import allAction from '../../redux/actions';
-import { useEffect, useState } from 'react'
+import { useCallback, useEffect, useState } from 'react'
 import { Post } from '../../types/Post'
 import PostItem from './PostItem'
 
 const PostList = () => {
-  const fetchedPosts = useSelector((state:any) => state.postList);
-  const dispatch = useDispatch();
-
-  useEffect(() => {
-    dispatch(allAction.loadPostList());
-  }, []);
-
   const [pageCount, setPageCount] = useState(0)
-  const [fetching, setFetching] = useState(false)
-  const [posts, setPosts] = useState<Post[]>([])
+  const [fetchingPost, setFetchingPost] = useState(false)
 
   const MAXIMUM_PAGES = 10
 
-  const handleScroll = () => {
-    const scrollHeight = document.documentElement.scrollHeight
-    const scrollTop = document.documentElement.scrollTop
-    const clientHeight = document.documentElement.clientHeight
+  const fetchedPosts = useSelector((state:any) => state.postList);
+  const dispatch = useDispatch();
 
+  const handleScroll = () => {
+    const {scrollHeight, scrollTop, clientHeight } = document.documentElement;
     if (scrollTop + clientHeight + 100 >= scrollHeight) {
-      fetchMoreData()
+      setFetchingPost(true)
     }
   }
 
-  window.addEventListener('scroll', handleScroll)
-  useEffect(() => {
-    return () => {
-      window.removeEventListener('scroll', handleScroll)
-    }
-  }, [])
-
-  const appendData = () => {
-    setFetching(true)
-    setPosts(fetchedPosts)
+  const fetchArticle = useCallback(()=> {
+    dispatch(allAction.loadPostList());
     setPageCount(pageCount + 1)
-    setFetching(false)
-  }
+    setFetchingPost(false)
+  },[pageCount])
```

Comment:
콤마를 쓰게 되었을 때 공백이 있는 것이 좋을 것 같습니다. 
```suggestion
  } ,[pageCount])
```

Eslint에 [Link](https://seungtaek-overflow.tistory.com/8)와 같이 규칙을 저희끼리 정하여 설정해주는 것이 좋을 것 같습니다!